File: backend/team6/src/main/java/programmers/team6/domain/member/entity/MemberInfo.java
package programmers.team6.domain.member.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AccessLevel;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import programmers.team6.global.entity.BaseEntity;

@Entity
@Getter
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class MemberInfo extends BaseEntity {

	@Id
	@Column(name = "member_info_id")
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;

	@Column(nullable = false)
	private String birth;

	@Column(nullable = false)
	private String email;

	@Column(nullable = false)
	private String password;

	@Builder
	public MemberInfo(String birth, String email, String password) {
		this.birth = birth;
		this.email = email;
		this.password = password;
	}
}

File: backend/team6/src/main/java/programmers/team6/domain/member/entity/MemberInfo.kt
package programmers.team6.domain.member.entity

import jakarta.persistence.*
import programmers.team6.global.entity.BaseEntity

@Entity
class MemberInfo (

    @field:Column(nullable = false)
     val birth: String,

    @field:Column(nullable = false)
    val email: String,

    @field:Column(nullable = false)
    val password: String
) : BaseEntity() {
    @Id
    @Column(name = "member_info_id")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private var id: Long? = null


}